File: cs 1400/In Class/Card/Card.java
/************************

Justin and Mark
CS 1400
Classwork

***********************/
enum Suit {DIAMONDS, HEARTS, CLUBS, SPADES}
			
enum Rank {DEUCE, THREE, FOUR, FIVE, SIX, SEVEN, EIGHT, NINE, TEN, JACK, QUEEN, KING, ACE}

public class Card
{
	private Rank rank;
	private Suit suit;
	
	public Card (Rank r, Suit s)
	{
		rank = r;
		suit = s;
	}
				
		@Override 
		public String toString()
		{
			return String.format("%s of %s",rank, suit);   
		}	
			
	//}//end main
	
}//end class	

File: cs 2530/In class work/Labs/Card/Card/Card.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Card
{
    enum Rank { Deuce, Three, Four, Five, Six, Seven, Eight, Nine, Ten, Jack, Queen, King, Ace }
    enum Suite { Spades = 9824, Clubs = 9827, Hearts = 9829, Diamonds = 9830 }
    struct Card
    {
        //auto implemented properties
        public Suite Suite { get; private set; }
        public Rank Rank { get; private set; }
        //constructor
        public Card( Suite mySuite, Rank myRank) : this()
        {
            Suite = mySuite;
            Rank = myRank;
        }
        
        public override string ToString()
        {
            return string.Format("{0} {1}", (char)Suite, Rank);
        }
    }
}
